### PR TITLE
use one single place for xfstests tool to avoid space is full issue

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: lisa/tools/xfstests.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ class Xfstests(Tool):
 
     @property
     def command(self) -> str:
-        return "xfstests"
+        return str(self._code_path)
```
Comment (squirrelsc) [inline lisa/tools/xfstests.py]: It looks is the folder "/xfstest-dev" does it work as the command here?

Comment (LiliDeng) [reply lisa/tools/xfstests.py]: command is not used in xfstests at all, change into 'f"ls -lt {self._code_path}"'?

Comment (squirrelsc) [reply lisa/tools/xfstests.py]: ok, so it looks like used for exists check only. If so, the code is ok. But you can add some comments to explain the weird logic.

Comment (LiliDeng) [reply lisa/tools/xfstests.py]: updated

File: lisa/tools/xfstests.py [hunk 2]
```diff
@@ -114,7 +114,8 @@ class Xfstests(Tool):
 
     @property
     def command(self) -> str:
-        return "xfstests"
+        # command is not used in xfstests
```
Comment (squirrelsc) [inline lisa/tools/xfstests.py]: "command is not used in xfstests" it's not true. The command is not directly to be called. But it's used to check if the command installed or not.

Comment (LiliDeng) [reply lisa/tools/xfstests.py]: in this tool, _check_exists is overwrite, I don't think command is used.

Comment (squirrelsc) [reply lisa/tools/xfstests.py]: If it's not used. set it a value like "mockup" and add the comments. It isn't confused.

Comment (LiliDeng) [reply lisa/tools/xfstests.py]: update

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**